README.md

```diff
@@ -1,2 +1,22 @@
 # hw01
-Repository for first CFSS homework assignment
+Repository for first CFSS homework assignment   
+
+## HELLO MY NAME IS...   
+
+Nicholena Honors
+
+*this stuff is tough*  
+
+**but I'm getting by**
+
+### I have a ton more to learn clearly...  
+
+#For breakfast I had...
+
+* [TEA](http://g3yxi3w953w3xjjaj1xcbuay.wpengine.netdna-cdn.com/wp-content/uploads/2015/01/Dollarphotoclub_68652169-e1422257422160.jpg)
+* [EGGS](http://www.incredibleegg.org/wp-content/themes/incredibleegg/assets/images/facts-left-egg.png)
+
```

Comment:
I love the picture links.  Another route would have been to include the pictures within the file.